cli/internal/run/run.go

```diff
@@ -225,7 +225,7 @@ func (c *RunCommand) Run(args []string) int {
 	rs := &runSpec{
 		Targets:      targets,
 		FilteredPkgs: filteredPkgs,
-		Opts:         runOptions,
+		Opts:         opts,
```

Comment:
Our option boundaries between `c` (`RunCommand`) and `rs` (`RunSpec`) here are a little weird to me. Just noting, not a request for change.

Reply:
`RunCommand` is not long for this world. The next PR in this series turns it into a wrapper around a `cobra` command.